gce/sysprep/instance_setup.ps1

```diff
@@ -286,9 +243,12 @@ function Enable-RemoteDesktop {
       services.
```

Comment:
If you are removing the spaces in between function docs in other files, you should do it everywhere.